BrainPortal/app/views/users/new.html.erb

```diff
@@ -76,6 +76,8 @@
     <p><label for="no_password_reset_needed">No need to reset initial password:</label>
     <%= check_box_tag :no_password_reset_needed, "1", params[:no_password_reset_needed] == "1" %>
 
+    <%= hidden_field_tag :signup_id, params[:signup_id] %>
```

Comment:
It is used in the `new` action so that it is preserved when the form fails to validate